README.md

```diff
@@ -1,32 +1,169 @@
 [![PyPI - Downloads](https://img.shields.io/pypi/dm/promptflow)](https://pypi.org/project/promptflow/)
 [![License: MIT](https://img.shields.io/github/license/microsoft/promptflow)](https://github.com/microsoft/promptflow/blob/main/LICENSE)
+[![](https://dcbadge.vercel.app/api/server/bnXr6kxs?compact=true&style=flat)](https://discord.gg/bnXr6kxs)
+
+> Welcome to join us to make Prompt flow!
 
-> Welcome to join us to make Prompt flow better by
-> participating [discussions](https://github.com/microsoft/promptflow/discussions),
-> opening [issues](https://github.com/microsoft/promptflow/issues/new/choose),
-> submitting [PRs](https://github.com/microsoft/promptflow/pulls).
+[Documentacion](https://microsoft.github.io/promptflow/) • [Issues](https://github.com/microsoft/promptflow/issues/new/choose)  • [Discord](https://discord.gg/bnXr6kxs) •  [Discussions](https://github.com/microsoft/promptflow/discussions) • [Contribute PRs](https://github.com/microsoft/promptflow/pulls).
 
 **Prompt flow** is a suite of development tools designed to streamline the end-to-end development cycle of LLM-based AI applications, from ideation, prototyping, testing, evaluation to production deployment and monitoring. It makes prompt engineering much easier and enables you to build LLM apps with production quality.
 
 With prompt flow, you will be able to:
 
-- Create executable workflows that link LLMs, prompts, Python code and other tools together.
-- Debug and iterate your flows, especially the interaction with LLMs with ease.
-- Evaluate your flow's quality and performance with larger datasets.
-- Integrate the testing and evaluation into your CI/CD system to ensure quality of your flow.
-- Deploy your flow to the serving platform you choose or integrate into your app's code base easily.
+- **Create and Iteratively Develop Flow**
+    - Create executable workflows that link LLMs, prompts, Python code and other tools together.
+    - Debug and iterate your flows, especially the interaction with LLMs with ease.
+- **Evaluate Flow Quality and Performance**
+    - Evaluate your flow's quality and performance with larger datasets.
+    - Integrate the testing and evaluation into your CI/CD system to ensure quality of your flow.
+    - Deploy your flow to the serving platform you choose or integrate into your app's code base easily.
 - (Optional but highly recommended) Collaborate with your team by leveraging the cloud version of [Prompt flow in Azure AI](https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/overview-what-is-prompt-flow?view=azureml-api-2).
 
-## Get Started with Prompt flow ⚡
+### Concept Overview
+
+[![concept](examples/tutorials/quick-start/media/concept.png)](https://microsoft.github.io/promptflow/concepts/concept-connections.html)
+
+------
+
+## Installation
+
+Ensure you have a python environment, `python=3.9` is recommended.
+
+```sh
+pip install promptflow promptflow-tools
+```
+
+## Quick Start ⚡
```

Comment:
we should have some of  these to  src/promptflow/README.md

which will be displayed here: https://pypi.org/project/promptflow/